### ISSUE TITLE
Qtip unpositioned for `mousover`

### ISSUE THREAD

**Issue by monprin** [cytoscape/cytoscape.js-qtip#21]
The new 2.5.1 version has not been working for me. it simply returns either a warning or an error stating:
```
Query.Deferred exception: Cannot assign to read only property 'left' of object '#<ClientRect>' 
TypeError: Cannot assign to read only property 'left' of object '#<ClientRect>'
```
This error was also in Firefox, which issued the error: 
```
jQuery.Deferred exception: setting a property that has only a getter z.reposition
```

2.4.0 worked well prior to this. Let me know if I can provide any other debugging information.
Thank you for all your work!
Joe

**Comment by narenana** [comment]
I get this only when using mouseover as trigger. works fine on tap.

**Comment by narenana** [comment]
@maxkfranz could you reproduce this?
I can share a fiddle to help.

**Comment by maxkfranz** [comment]
Patch released as 2.5.2
